composer.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "bshaffer/oauth2-server-bundle",
+    "name": "alpha1125/oauth2-server-bundle",
```

Comment:
You just do a new commit to revert that, and push it in your branch and the PR will be updated accordingly.